plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx

```diff
@@ -36,7 +35,6 @@ import ExternalLinkIcon from '@material-ui/icons/Launch';
 const useStyles = makeStyles(theme => ({
   root: {
     maxWidth: 720,
-    margin: theme.spacing(2),
```

Comment:
Makes me wonder, do people really like the maxWidth too? :) I know there's readability benefits with short lines but truncated/line-breaking things in tables aren't great either

Reply:
Yeah, great point.  If I take it out, it goes full width, and the table is sooooo small it's not really worth it.

Honestly, I think the table as a whole might be much better to be similar to an AboutCard and could be made more visual, but it just wouldn't be as simple as a table where we can add/remove columns.

So I think if it's OK for the purposes of this incremental update I won't touch it, but agree I think this entire page could do with some refactoring. Maybe check the GitHub Actions and CircleCI plugins and see if there could be some feature parity (and potentially reusable CI components) across them.

Reply:
Yep it was mostly a side note. Not a lot of other things use width-limiting.

Reply:
and oops, left my maxWidth testing in; reverted now